src/domain/resource/utils.js

```diff
@@ -401,7 +401,7 @@ export const isTimeRangeReservable = (resource, start, end) => {
 
   // Reservation cannot be longer than the resources max period if max period is set.
   const maxPeriod = get(resource, 'max_period', null);
-  if (maxPeriod) {
+  if (!isStaff && maxPeriod) {
```

Comment:
Please add comment for this exception: 
Something as `As a staff, user have permission to ...`

Reply:
I added the code comment to the function dockblock , that should be more correct right?